orbit/pkg/scripts/exec_nonwindows.go

```diff
@@ -4,15 +4,37 @@ package scripts
 
 import (
 	"context"
+	"fmt"
+	"os"
 	"os/exec"
 	"path/filepath"
+
+	"github.com/fleetdm/fleet/v4/server/fleet"
 )
 
 func execCmd(ctx context.Context, scriptPath string) (output []byte, exitCode int, err error) {
 	// initialize to -1 in case the process never starts
 	exitCode = -1
 
+	contents, err := os.ReadFile(scriptPath)
+	if err != nil {
+		return nil, -1, err
+	}
```

Comment:
not a big deal at all. Feels a bit odd to write the file in the caller and open that file here again + potentially modifying it's permissions.

I understand it's not a straightforward refactor tho.

Could you please update the comment here?

https://github.com/fleetdm/fleet/blob/fb1ba2ceaca216bb20f3759d66681fc05587b077/orbit/pkg/scripts/scripts.go#L111-L115

Reply:
Done 👍